packages/nimble-components/src/table-column/text/tests/table-column-text.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { html } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { Table } from '../../../table';
+import { TableColumnText } from '..';
+import { waitForUpdatesAsync } from '../../../testing/async-helpers';
+import { type Fixture, fixture } from '../../../utilities/tests/fixture';
+import type { TableRecord } from '../../../table/types';
+import { TablePageObject } from '../../../table/tests/table.pageobject';
+
+interface SimpleTableRecord extends TableRecord {
+    stringData: string | null;
+    numericData: number | null;
+    booleanData: boolean | null;
+    dateData: Date | null;
+}
+
+const simpleTableData = [
+    {
+        stringData: 'string 1',
+        numericData: 8,
+        booleanData: true,
+        dateData: new Date(2008, 12, 11)
+    },
+    {
+        stringData: '<button></button>',
+        numericData: 0,
+        booleanData: true,
+        dateData: new Date(2022, 5, 30)
+    },
+    {
+        stringData: null,
+        numericData: null,
+        booleanData: null,
+        dateData: null
+    }
+] as const;
+
+async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
+    return fixture<Table<SimpleTableRecord>>(
+        html`<nimble-table>
+                <${DesignSystem.tagFor(
```

Comment:
nit: You could consider assigning `DesignSystem.tagFor(TableColumnText)` to a variable to make this a bit more readable.

Reply:
Done.